apps/site/lib/site_web/channels/vehicle_channel.ex

```diff
@@ -51,7 +54,7 @@ defmodule SiteWeb.VehicleChannel do
           id: vehicle.id,
           icon: "vehicle-bordered-expanded",
           rotation_angle: vehicle.bearing,
-          shape_id: trip.shape_id,
+          shape_id: (trip && trip.shape_id) || nil,
```

Comment:
`trip && trip.shape_id` will return `nil` if `trip` is `nil`, so the ` || nil` is unecessary.

    iex(1)> trip = nil
    nil
    iex(2)> trip && trip.shape_id
    nil